chainer/functions/connection/n_step_lstm.py

```diff
@@ -483,13 +483,13 @@ def n_step_lstm(
 
         ys = []
         for x in xs:
-            batch = len(x.data)
+            batch = x.shape[0]
             h_next = []
             c_next = []
             for layer in six.moves.range(n_layers):
                 h = hx[layer]
                 c = cx[layer]
-                if len(h.data) > batch:
+                if h.shape[0] > batch:
```

Comment:
`len(h.shape)` is still valid. Is there any reason why change from `len()` to indexing?